src/config.ts

```diff
@@ -94,6 +94,18 @@ export interface IPermitConfig {
    * in the PDP cache before returning the response.
    */
   factsSyncTimeout: number | null;
+  /**
+   * Configures the Policy Decision Point (PDP) address.
+   */
+  opa?: string;
```

Comment:
Maybe call it `opaUrl` so it will be self-descriptive

Reply:
it equivalent to pdp if i called it opaUrl ,  than pdp (which is url) become a confuding name